web/csrf.md

```diff
@@ -58,3 +58,7 @@ In practice, Referer headers may be removed by the browser, the operating system
 _Further reading:_ [OWASP Cheat Sheet on CSRF](https://owasp.org/www-community/attacks/csrf)
 
 [^1]: Yes, the "Referer" field represents a roughly three decade old misspelling of referrer. This is a silly example of how "legacy", that is old design decisions, can impact things decades later because it can be very hard to change things
+
+## 21.4. Defense: SameSite Cookie Attribute
+
+A last way to defend against CSRF attacks is to add a flag to cookies that specifies it should be sent only when the domain of the cookie exactly matches the domain of the origin. For example, with this flag, if `evil.com` causes the browser to make a request to `bank.com`, cookies for `bank.com` will not be sent if because the origin domain (`evil.com`) and cookie domain (`bank.com`) are different. Unfortunately, this browser-side defense is not implemented on all browsers, thus is usually only used as a defense-in-depth strategy.
```

Comment:
SameSite actually works with "sites" which are coarser than origins.  For instance, usercontent.berkeley.edu can attack www.berkeley.edu.  But maybe that's unimportant in this context.